R/load.R

```diff
@@ -729,15 +730,13 @@ renv_load_quiet <- function() {
   config$startup.quiet(default = default)
 }
 
-renv_load_finish <- function(project, lockfile) {
+renv_load_finish <- function(project = NULL, lockfile = NULL) {
 
   renv_project_set(project)
-  renv_load_check(project)
 
-  if (!renv_load_quiet()) {
-    renv_load_report_project(project)
-    renv_load_report_python(project)
-  }
+  renv_load_check(project)
+  renv_load_report_project(project)
+  renv_load_report_python(project)
 
   if (interactive()) {
```

Comment:
Is it really the case we should only do this reports interactively? If so, should `renv_snapshot_auto_update()` also only be done interactively? The current behaviour seems inconsistent to me.

Reply:
The intention was for automatic snapshots to only happen in interactive sessions, so I'm happy to make things more consistent here.